go.mod

```diff
@@ -95,6 +95,7 @@ require (
 	google.golang.org/grpc v1.54.0
 	google.golang.org/protobuf v1.30.0
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
I don't think we need both `yaml.v2` and `yaml.v3`. The `yaml.v2` package is imported by `internal/audit/conf.go`. Let's change it to import `yaml.v3`.